examples/simple/app/views/welcome/index.html.erb

```diff
@@ -1,2 +1 @@
-<h1>Welcome#index</h1>
-<p>Find me in app/views/welcome/index.html.erb</p>
+<%= render_react_component('MyComponent.js', :name => 'Hypernova') %>
```

Comment:
for the next minor of hypernova-ruby, we should try to come up with a way so that the ERB isn't needed at all :-)